database/metadata/databases/default/tables/lookups_agency.yaml

```diff
@@ -9,13 +9,6 @@ array_relationships:
         table:
           name: crashes_cris
           schema: public
-  - name: crashes_edits
-    using:
-      foreign_key_constraint_on:
-        column: investigat_agency_id
-        table:
-          name: crashes_edits
-          schema: public
   - name: crashess
```

Comment:
not related to this PR, because its in production but it just jumped out to me here, why is this named `crashess`, its like this in all of the lookups relationships

Reply:
related—`units_cri`: https://github.com/cityofaustin/vision-zero/pull/1495#discussion_r1701899781

Reply:
thanks for flagging it—let's leave it as-is for now. fortunately, we're not using these array relationships afaik so it won't be too painful to change.

Reply:
it makes it feel more magnificent. countess, duchess, crashess